styles.css

```diff
@@ -340,4 +340,83 @@ main {
   grid-template-columns: 20% 80px;
 }
 
-/* } */
+footer {
+  background-color: #6070ff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-top-left-radius: 5rem;
+  row-gap: 20px;
+  padding: 7.5rem 0;
+  width: 100%;
+  background-image: url(images/contact-mobile-1.png);
+  background-repeat: no-repeat;
+  background-size: 110%;
+}
+
+.contact-me {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+}
+
+.footer-p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin-left: 10%;
+}
+
+.feedback {
+  display: grid;
+  grid-template-columns: auto;
+  row-gap: 20px;
+  width: 447px;
+}
```

Comment:
- [ ] Kindly fix the overflow of your form, I would highly recommend removing the fixed width from `.feedback` and the error should be fixed 🙏 

![HINT12](https://user-images.githubusercontent.com/89282221/200801446-09aba725-05d6-49c6-8ec4-41b43450e4b7.gif)